phabfive/maniphest.py

```diff
@@ -52,10 +53,24 @@ def create_from_config(self, config_file, dry_run=False):
             data = yaml.load(stream, Loader=yaml.Loader)
 
         #####
-        # STEP: Data structure validation, check title + description, projects existance
+        # STEP: Data structure validation, check title + description, projects, owner, subscribers existance
 
         ### Validate all projects is correct data types and exists in Phabricator instance
-        projects_query = self.phab.project.search(constraints={"name": ""})
+        #create a new construct to consolidate all project/tags
+        projects_query = {}
+        projects_query["data"] = []
+
+        r = self.phab.project.search(constraints = {"name": ""})
+        length = len(r.data)
+        a = r['cursor']['after']
+        projects_query["data"].extend(r.data)
+
+        # Added for loop to handle phab instances with more than 100 project/tags
+        while (length >= 100 and a is not None):
+            r = self.phab.project.search(constraints = {"name": ""}, after=a)
+            length = len(r.data)
+            a = r['cursor']['after']
+            projects_query["data"].extend(r.data)
```

Comment:
There is still single variable names here that we do not want in, at minimum variables should be 3 letters or more. Please update this through out the entire PR